www/js/app.js

```diff
@@ -994,7 +996,7 @@ define(['jquery', 'zimArchiveLoader', 'uiUtil', 'settingsStore','abstractFilesys
         for (var i = 0; i < archiveDirectories.length; i++) {
             var archiveDirectory = archiveDirectories[i];
             if (archiveDirectory === "/") {
-                alert("It looks like you have put some archive files at the root of your sdcard (or internal storage). Please move them in a subdirectory");
+                uiUtil.systemAlert("Alert", "It looks like you have put some archive files at the root of your sdcard (or internal storage). Please move them in a subdirectory", false);
             }
             else {
```

Comment:
Not your style as it was in the original, but here you might as well tidy this up by using the more concise and typographically pleasing `} else {` style.

Reply:
> @gaurav7019 The reason the app won't even load in IE11 is because of the syntax you used in `uiUtil.js` which is incompatible with IE11. See the console image below. IE11 thinks the main function there has incorrect syntax, so uiUtil.js fails to load entirely. The relevant error is `Expected ')'`. Just use the compatible syntax I suggested for setting default values.
> 
> ![image](https://user-images.githubusercontent.com/4304337/151940073-6234fabc-9500-4a17-90dc-e6ed4aec43bd.png)

Sure, thanks for the info